crates/bevy_ecs/src/system/query.rs

```diff
@@ -737,7 +737,7 @@ impl<'w, 's, D: QueryData, F: QueryFilter> Query<'w, 's, D, F> {
         &self,
     ) -> QueryCombinationIter<'_, 's, D, F, K> {
         // SAFETY:
-        // - `self.world` has permission to access the required components.
+        // - `self.world` has permission to access the related components.
```

Comment:
Same as above: this doesn't appear to be related to required/related components, I think this is just a find&replace oopsie!